templates/aws-tap-entrypoint-existing-vpc.template.yaml

```diff
@@ -802,6 +802,9 @@ Resources:
         - S3Bucket: !If [UsingDefaultBucket, !Sub '${QSS3BucketName}-${AWS::Region}', !Ref QSS3BucketName]
           S3Region: !If [UsingDefaultBucket, !Ref AWS::Region, !Ref QSS3BucketRegion]
       Parameters:
+        QSS3BucketName: !Ref QSS3BucketName
```

Comment:
if this is already taken care of in the upstream repo then we can get it by the next merge/pull from upstream.

Reply:
This came in, as a cherry-pick, from the upstream repo / commit.
I don't see a problem if bring it in, and merge it; makes the diff between our `main` and the upstream a little smaller.
Am I missing what you are concerned about?